src/main/java/com/sprint/mission/discodeit/Enum/ChannelType.java

```diff
@@ -0,0 +1,6 @@
+package com.sprint.mission.discodeit.Enum;
```

Comment:
패키지명은 보통 소문자로 짓는게 관례예요. 

```suggestion
package com.sprint.mission.discodeit.enum;
```

Reply:
- 참고문서: [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names)

> Package and module names use only lowercase letters and digits (no underscores).

패키지 및 모듈 이름은 소문자와 숫자만 사용합니다(밑줄은 사용하지 않습니다).